apps/storefront/src/pages/MyOrders/index.test.tsx

```diff
@@ -307,6 +307,46 @@ describe('when a personal customer', () => {
       expect(orderHeader).toHaveAttribute('aria-sort', 'descending');
     });
 
+    it('toggles sort order when clicking the current sort column', async () => {
+      const getOrders = vi.fn().mockReturnValue(buildGetCustomerOrdersWith('WHATEVER_VALUES'));
+
+      server.use(
+        graphql.query('GetCustomerOrders', ({ query }) => HttpResponse.json(getOrders(query))),
+      );
+
+      renderWithProviders(<MyOrders />, { preloadedState });
+
+      await waitForElementToBeRemoved(() => screen.queryAllByRole('progressbar'));
+
+      const createdOnHeader = screen.getByRole('columnheader', { name: 'Order' });
+
+      expect(createdOnHeader).toHaveAttribute('aria-sort', 'descending');
+
+      when(getOrders)
+        .calledWith(stringContainingAll('orderBy: "bcOrderId"'))
+        .thenReturn(
+          buildGetCustomerOrdersWith({
+            data: {
+              customerOrders: {
+                totalCount: 1,
+                edges: [buildCustomerOrderNodeWith({ node: { orderId: '66996' } })],
```

Comment:
might be a bit more intuitive to have two order nodes returned by each mock (but in different respective orders)

we would be mocking out BE responsibilities, but it gives readers a clearer sense of user expectations 🍹 

also, are people using order by ID as a stand in for ordering by created date? Or does it solve for some other use case?

Reply:
> might be a bit more intuitive to have two order nodes returned by each mock (but in different respective orders)

can do.

> also, are people using order by ID as a stand in for ordering by created date? Or does it solve for some other use case?

I don't know really, we offer both, you can sort by `orderId`, and `createdAt` 🤷